### ISSUE TITLE
Attempted import error: 'useSubscription' is not exported from 'mqtt-react-hooks'.

### ISSUE THREAD

**Issue by mrzachsnyder** [VictorHAS/mqtt-react-hooks#2]
Hello VictorHAS! First of all, thank you very much for this repo, so incredibly useful.

tl;dr I'm getting this error when I call useSubscription().
```
./src/components/App.js
Attempted import error: 'useSubscription' is not exported from 'mqtt-react-hooks'.
```

I just got this app spun up and started messing with your library so everything in question is happening within App.js, which I've included below in its entirety.

As you can see I got useSubscribeState() to work, but it returns an entire array of all the MQTT messages. So impressive but too much! **All I want is the latest message in [messages].** I suspect that useSubscription() will meet my needs but I'm running into that import error, which I thought may be useful to bring up here. Any and all advice/feedback is extremely welcomed - as you can tell I'm pretty new to React. Thank you very much!

```
import React from 'react';
import './App.css';

import { useMqttState, useSubscribeState } from 'mqtt-react-hooks';
import { useSubscription } from 'mqtt-react-hooks';

const App = () => {
  const { mqtt } = useMqttState();
  const { messages } = useSubscribeState();

  const sendMessage = (message) => {
    console.log('Sent over MQTT: ' + message);
    return mqtt.publish('@', message);
  }

  // const test = useSubscription('leapCurrentSpeed');
  // console.log(test);

  const updateSpeed = (messages) => {
    if (messages !== undefined && messages.length !== 0) {
      console.log(messages);
      // Sketchy way of getting to the last message in [messages]
      const messagesLength = messages.length;
      const message = JSON.parse(messages[messagesLength - 1].message);
      if (message.hasOwnProperty('currentSpeed')) {
        const { currentSpeed } = message;
        return currentSpeed;
      }
    }
    else {
      const currentSpeed = 100;
      // console.log('in else statement');
      return currentSpeed;
    }
  }

  return (
    <div className="App">
      <button onClick={() => sendMessage('test message')}>
        Send MQTT message
      </button>
      <div className="speedReading">
        {updateSpeed(messages)}
      </div>
    </div>
  )
}

export default App;
```

**Comment by VictorHAS** [comment]
Hi @mrzachsnyder ! I am honored to have liked the repository, I ask if you would like to share your knowledge in this repo, you need not be afraid. I am new to react too and I decided to make this lib because I really needed it and I didn't see any good ones. So I'm trying to create one, but my concepts are not very advanced.

First: Which version are you using? useSubscription is only available in version 1.1.1, and I removed useSubscribeState. You can see this on example folder

Unfortunately I haven't figured out how to match wildcards yet, but in relation to receiving the last message I can make it return on useSubscription

**Comment by mrzachsnyder** [comment]
Thank you so much for the quick response! You are exactly right, can't believe it was that easy.
```
    "mqtt-react-hooks": "^1.0.3",
```

We moved the project into plain html and js instead of React (I'm the only React guy on our dev team right now), but I am going to revisit this as soon as we finish our current dev effort. Very excited to update to latest version and try useSubscription(). More to come - feel free to close the issue.
